Clients/src/Swag4Net.Core/ApiModel.fs

```diff
@@ -0,0 +1,113 @@
+namespace Swag4Net.Core
+
+open System.Net
+
+module ApiModel =
+  type TypeName = string
+  type Any = string
+  type RegularExpression = string
+
+  type Api =
+    { Infos:Infos
+      Host:string
+      BasePath:string
+      Routes:Route list
+      Definitions:Schema list }
+  and Infos =
+    { Description:string
+      Version:string
+      Title:string
+      TermsOfService:string
+      Contact:Contact option
+      License:License option }
+  and Contact = 
+    | Email of string
+  and License = 
+    { Name:string; Url:string }
+  and Schema =
+      {
+        Type: DataType
+        Title: string option
+        AllOf: Schema option
```

Comment:
AllOf is a list

Reply:
En effet, comme les autres xxOf d'ailleurs